lxc/cluster.go

```diff
@@ -128,6 +128,7 @@ func (c *cmdClusterList) Run(cmd *cobra.Command, args []string) error {
 		i18n.G("DATABASE"),
 		i18n.G("STATE"),
 		i18n.G("MESSAGE"),
+		i18n.G("ARCH"),
```

Comment:
I think we'll want `ARCHITECTURE` for that one, I've sent a branch to make `lxc image list` consistent with `lxc list`.